migrations/sql/V2020.04.08.15.06__add_new_fields_to_now_document_templates.sql

```diff
@@ -0,0 +1,352 @@
+UPDATE document_template SET form_spec_json = '[
```

Comment:
Is this entire file necessary? Or are the equivalent updates in seed_data enough?..

Reply:
The inserts in seed data typically have "ON CONFLICT DO NOTHING", which for our intents means if the unique key already exists do nothing. It doesn't handle updating any fields if they have already been inserted. IMO the seed data file shouldn't exist, but it can be nice to have to review at times.

Reply:
I think because seed data runs last, this file might not be needed, as seed data will insert these values.

Reply:
If these are net new, the files is not needed. If they existed and you are updating them, they are needed.

Reply:
Thanks for clarifying. 😄